### PR TITLE
Adding Spanish language support to app

### PR DESCRIPTION
**Pull request by zydeico**
I added Spanish support to app, I follow your Readme.md steps and you can verify all translations :)

### CODE REVIEW

File: reminders-menu-bar.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -641,7 +646,7 @@
 				COMBINE_HIDPI_IMAGES = YES;
 				DEAD_CODE_STRIPPING = YES;
 				DEVELOPMENT_ASSET_PATHS = "\"reminders-menu-bar/Preview Content\"";
-				DEVELOPMENT_TEAM = 3398VKLH28;
+				DEVELOPMENT_TEAM = "";
```
Comment (DamascenoRafael) [inline reminders-menu-bar.xcodeproj/project.pbxproj:649]:
```suggestion
				DEVELOPMENT_TEAM = 3398VKLH28;
```

Reverting change as it is not related to the addition of the new language.

File: reminders-menu-bar.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -653,6 +658,7 @@
 				MARKETING_VERSION = 1.8.0;
 				PRODUCT_BUNDLE_IDENTIFIER = "br.com.damascenorafael.reminders-menu-bar";
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				PROVISIONING_PROFILE_SPECIFIER = "";
```
Comment (DamascenoRafael) [inline reminders-menu-bar.xcodeproj/project.pbxproj:661]:
```suggestion
```

Reverting change as it is not related to the addition of the new language.

File: reminders-menu-bar.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -679,6 +685,7 @@
...
```
Comment (DamascenoRafael) [inline reminders-menu-bar.xcodeproj/project.pbxproj:688]:
```suggestion
```

Reverting change as it is not related to the addition of the new language.

File: reminders-menu-bar.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -667,7 +673,7 @@
 				COMBINE_HIDPI_IMAGES = YES;
 				DEAD_CODE_STRIPPING = YES;
 				DEVELOPMENT_ASSET_PATHS = "\"reminders-menu-bar/Preview Content\"";
-				DEVELOPMENT_TEAM = 3398VKLH28;
+				DEVELOPMENT_TEAM = "";
```
Comment (DamascenoRafael) [inline reminders-menu-bar.xcodeproj/project.pbxproj:676]:
```suggestion
				DEVELOPMENT_TEAM = 3398VKLH28;
```

Reverting change as it is not related to the addition of the new language.

File: reminders-menu-bar/Resources/es-419.lproj/Localizable.strings [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+"renameReminderAlertTitle" = "Cambiar nombre";
+"renameReminderAlertMessage" = "Esta acción cambiará el nombre del recordatorio y no se puede deshacer";
+"renameReminderAlertConfirmButton" = "Guardar";
+"renameReminderAlertCancelButton" = "Cancelar";
+"removeReminderOptionButton" = "Eliminar";
+"removeReminderAlertTitle" = "¿Eliminar recordatorio?";
+"removeReminderAlertMessage" = "Esta acción eliminará '%@' y no se puede deshaser";
+"removeReminderAlertConfirmButton" = "Eliminar";
+"removeReminderAlertCancelButton" = "Cancelar";
+"reminderMoveToMenuOption" = "Mover a...";
```
Comment (DamascenoRafael) [inline reminders-menu-bar/Resources/es-419.lproj/Localizable.strings:14]:
```suggestion
"reminderMoveToMenuOption" = "Mover a ...";
```

Adding space before punctuation to be concise with other languages.

**Review by DamascenoRafael — CHANGES_REQUESTED** [review]
This is awesome! ✨  I left a few suggestions for changes so we can merge it

File: reminders-menu-bar/Resources/es-419.lproj/Localizable.strings (hunk 5 above)
Comment (zydeico) [reply reminders-menu-bar/Resources/es-419.lproj/Localizable.strings]: @DamascenoRafael the grammar of Spanish is that the ellipses are together in the word :D
